serverlessworkflow/modules/ROOT/pages/cloud/deploying-on-minikube.adoc

```diff
@@ -1 +1,369 @@
+----
+alias kubectl="minikube kubectl --"
+----
+====
+
+Now, the `kubectl` command should be available.
+
+=== Installing Knative on Minikube
+.Procedure
+. Install and configure Knative on Minikube
++
+Knative CLI offers a plugin called `quickstart` that will configure everything that is needed for you, to install it follow this link:{kn_cli_quickstart_plugin_url}[steps].
++
+With the plugin configured, execute:
++
+[source,shell]
+----
+kn quickstart minikube
+----
++
+NOTE: To install everything manually, there is this cool link:{knative_on_minikube_step_by_step_url}[post] from Red Hat Developers blog.
++
+To open the Minikube WebConsole on the `knative` profile, use the following command
++
+[source,shell]
+----
+minikube dashboard --profile knative
+----
+The command above will open the Kubernetes Management Console on your browser, if that does not open, copy the `url` that was returned.
+
+
+== Deploying a Serverless Application on Minikube
+
+At this point, is expected that you:
+
+* Followed the previous steps to install Minikube and Knative.
+* Have the CLI tools properly installed.
+* Have a Serverless Application Container already built.
+
+[IMPORTANT]
+.Configure Docker to use the in-cluster Docker daemon
+====
+With Minikube, you can share your local registry with it, to do this execute this command:
+
+[source,shell]
+----
+eval $(minikube -p minikube docker-env --profile knative
+----
+
+Note that, if the Application Container was built before configuring it, you might be required to build again to make the image available onthe Minikube Docker.
+
+Be sure to use the following system property when building native container images using Minikube's Docker Daemon:
+[source,shell]
+----
+-Dquarkus.native.remote-container-build=true
+----
+
+Note that, to make Knative able to fetch images from Minikube's Docker daemon you might be required to tag the images with some one of these registry options:
+----
+- ko.local
+- dev.local
+----
+To do this use the `-Dquarkus.container-image.registry=some_of_the_values_above`. For more information please visit this link:{knative_issue_url}[link].
+Another important thing to remember is that, you might need to set the `imagePullPolicy` to `Never` or `IfNotPresent` otherwise Minikube will try to pull it from a remote registry, this behavior can be avoided by tagging the image with some domains listed above.
+====
+
+For this tutorial we will use the Application used on the xref:cloud/build-workflow-image-with-quarkus-cli.adoc[Building Serverless Workflow Images using Quarkus CLI] guide.
+
+=== Preparing the environment
+
+Before start the procedure to install de Serverless Application, make sure you have started the `minikube tunnel` pointing to the `knative` profile.
+
+.Procedure
+. Start the Minikube tunnel using Knative profile
++
+[source,shell]
+----
+minikube tunnel --profile knative
+----
++
+NOTE: The tunnel command will start in a loop, it must be running during all this guide.
```

Comment:
Please add a note that this command may require the user's password. It happened to me before. You start the command, think it's ready, and there's this tiny message saying "password" that you won't notice quickly.

Reply:
It happens only on Mac and Windows, right?